config/database.yml.example

```diff
@@ -2,11 +2,10 @@ default: &default
   adapter: postgresql
   encoding: unicode
   pool: 5
-  username:
-  password:
-  # If you using docker the host should be "db"
-  host: localhost
-  port: 5432
+  username: <%= ENV.fetch('POSTGRES_USER', '') %>
+  password: <%= ENV.fetch('POSTGRES_PASSWORD', '') %>
+  host: <%= ENV.fetch('POSTGRES_HOST', 'localhost') %>
+  port: <%= ENV.fetch('POSTGRES_PORT', '5432') %>
```

Comment:
Is it possible to avoid all these changes if we pass `DATABASE_URL` env variable instead of all these different vars?

Reply:
1) Thanks for taking the time to test this, I'll look into that.
2) I think a `DATABASE_URL` will still be composed by these values, if we use that kind of thing for the template then I think we should change it so we have consistency through out the code base.

Reply:
I think Rails gives more importance to `DATABASE_URL`. If it is defined, Rails will ignore the `database.yml` config, that's why I suggested it, just to simplify all the changes. https://edgeguides.rubyonrails.org/configuring.html